packages/react-i18n/src/i18n.ts

```diff
@@ -129,7 +142,12 @@ export default class I18n {
       };
     }
 
-    return translate(id, normalizedOptions, this.translations, this.locale);
+    try {
+      return translate(id, normalizedOptions, this.translations, this.locale);
+    } catch (error) {
+      this.onError(error);
+      return '';
```

Comment:
Should we make the consumer supply a component for this? Maybe some sort of empty state is better than the empty string? Like a line that looks like it never loads?

Reply:
I think this gets really tricky because then you probably want a bunch of context about the type of error and other info. These errors should be rare and dealt with immediately, so I think this is enough to get that done.